packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -42,21 +42,19 @@ export class Span implements types.Span {
   constructor(
     parentTracer: types.Tracer,
     spanName: string,
-    traceId: string,
+    parentSpanContext: SpanContext,
```

Comment:
Why not simply pass the actual `SpanContext` that will be used at that point? Meaning it's not the parent that is passed, but you are basically passing the context that the span will itself use directly. If you don't have a parent span ID, I feel it's incorrect to pass it as being a parent.

Reply:
Hmm, which means `spanId` should be created by the tracer and we need to pass parentSpanId seperately (as a optional param) right?

Reply:
After thinking it over it makes sense to me to pass actual `SpanContext`. ptal again.

Reply:
The new implementation is not actually what I had in mind, but in any case this is an implementation detail that can be changed, so let's go forward with this for now.